runtime/Cpp/runtime/src/misc/IntervalSet.cpp

```diff
@@ -13,52 +13,31 @@
 using namespace antlr4;
 using namespace antlr4::misc;
 
-IntervalSet const IntervalSet::COMPLETE_CHAR_SET = []() {
-  IntervalSet complete = IntervalSet::of(Lexer::MIN_CHAR_VALUE, Lexer::MAX_CHAR_VALUE);
-  complete.setReadOnly(true);
-  return complete;
-}();
+IntervalSet const IntervalSet::COMPLETE_CHAR_SET = 
+    IntervalSet::of(Lexer::MIN_CHAR_VALUE, Lexer::MAX_CHAR_VALUE);
 
-IntervalSet const IntervalSet::EMPTY_SET = []() {
-  IntervalSet empty;
-  empty.setReadOnly(true);
-  return empty;
-}();
+IntervalSet const IntervalSet::EMPTY_SET;
 
-IntervalSet::IntervalSet() {
-  InitializeInstanceFields();
+IntervalSet::IntervalSet() : _intervals() {
 }
 
-IntervalSet::IntervalSet(const std::vector<Interval> &intervals) : IntervalSet() {
-  _intervals = intervals;
+IntervalSet::IntervalSet(const IntervalSet &set) : _intervals(set._intervals) {
```

Comment:
I don't like the changes to the c-tors. I took care to use delegating c-tors everywhere and want to stay with them, instead of repeating init of internal values over and over again. Please revert.

Reply:
Can you pls change this back also? As I said already, I like to stay with delegating c-tors.

Reply:
OK